src/CharacterList.js

```diff
@@ -57,14 +90,19 @@ class CharacterList extends Component {
         <ul className="nav_items">
           {alphabet.map((letter, index) => <AlphaLinks letter={letter} key={index} selected={this.state.letter === letter} />)}
         </ul>
-        {!this.state.characters.length && <Loader />}
+        {!this.state.total && <Loader />}
```

Comment:
Is there any potential for total to come back as zero or undefined from the fetch? if so, the loader won't disappear to display the error state